src/main/java/io/akka/tagging/api/TaggingEndpoint.java

```diff
@@ -70,6 +71,11 @@ private void startTaggingWorkflow(StartTaggingRequest startTagging, List<Integer
       .method(TaggingView::getCount)
       .invoke();
 
+    //keep the history of prompts
+    componentClient.forEventSourcedEntity("tagging-prompt")
+      .method(PromptTemplate::update)
+      .invoke(startTagging.prompt());
```

Comment:
You mentioned something about this, but what was the reason again?
I don't see that the PromptTemplate is used anywhere else?

Reply:
Yes, this is redundant, because I keep the prompt also in the workflow state. My motivation behind it was to have a nice history of prompt changes only. Many workflows might have the same prompt, so it's not that fast to check only the prompt changes. `PromptTemplate::update` ignores updates with the same prompt. If you think that this is confusing, I can remove it.